vnext/ReactUWP/Views/ShadowNodeBase.cpp

```diff
@@ -112,4 +130,74 @@ void ShadowNodeBase::UpdateTransformPS()
   }
 }
 
+void ShadowNodeBase::UpdateHandledKeyboardEvents(KeyboardType keyboardType, folly::dynamic const& value)
+{
+  if (keyboardType == KeyboardType::KeyDown) {
+    m_handledKeyDownKeyboardEvents = KeyboardHelper::FromJS(value);
+  }
+  else
+  {
+    m_handledKeyUpKeyboardEvents = KeyboardHelper::FromJS(value);
+  }
+
+  UpdateKeyboardEventHooks(m_view);
+}
+
+void ShadowNodeBase::UpdateKeyboardEventHooks(XamlView const& xamlView)
+{
+  auto shouldHook = ShouldHookKeyboardEvent(m_view);
+  if (shouldHook != m_keyboardEventHooked)
+  {
+    if (shouldHook && m_view)
```

Comment:
>m_view [](start = 22, length = 6)

why the null check?  Prefer an assert if this should never happen.

Reply:
agree.